### ISSUE TITLE
Trails of Cold Steel 2 Force Close

### ISSUE THREAD

**Issue by dvnfad18** [Vita3K/Vita3K-Android#455]
Cannot launch the game in the newest version of vita3k (v8) , it is force close over and over. Meanwhile the first series works perfectly

**Comment by Kodachiyo** [comment]
Same. I also heard from others that Cold Steel II used to run in previous versions (despite slowdowns depending on device) then started crashing at startup in the latest version.

**Comment by brandoswaggins** [comment]
> Same. I also heard from others that Cold Steel II used to run in previous versions (despite slowdowns depending on device) then started crashing at startup in the latest version.

Can confirm this is what started happening. Started the game two days before the update, rip my chapter 1 progress

**Comment by Kodachiyo** [comment]
> > Same. I also heard from others that Cold Steel II used to run in previous versions (despite slowdowns depending on device) then started crashing at startup in the latest version.
> 
> Can confirm this is what started happening. Started the game two days before the update, rip my chapter 1 progress

Let's hope it'll get fixed next update, just don't delete the Vita3k data. 

Was able to continue my progress in Cold Steel 1 from v7 in v8 this way.

**Comment by dvnfad18** [comment]
> > Same. I also heard from others that Cold Steel II used to run in previous versions (despite slowdowns depending on device) then started crashing at startup in the latest version.
> 
> Can confirm this is what started happening. Started the game two days before the update, rip my chapter 1 progress

Well u can copy ur sv data before downgrade it

**Comment by Kodachiyo** [comment]
> > > Same. I also heard from others that Cold Steel II used to run in previous versions (despite slowdowns depending on device) then started crashing at startup in the latest version.
> > 
> > 
> > Can confirm this is what started happening. Started the game two days before the update, rip my chapter 1 progress
> 
> Well u can copy ur sv data before downgrade it

Yep, but I did it solely for making sure it was because of versions. I was able to run it, but sadly v7 and older versions exhibit massive gradual slowdowns in any game. Thanks tho
